tests/integration/issues/github_1072/test_queryset.py

```diff
@@ -0,0 +1,43 @@
+import os
+import numpy as np
+
+from jina.proto import jina_pb2
+from jina.drivers.helper import array2pb
+from jina.flow import Flow
+
+cur_dir = os.path.dirname(os.path.abspath(__file__))
+
+
+def test_queryset_with_struct(tmpdir):
+    os.environ['JINA_TEST_QUERYSET_WORKSPACE'] = str(tmpdir)
```

Comment:
one suggestion, you can reuse the fixture what we have in `conftest.py`, to create a new fixture there, something like:
```python
@pytest.fixture(scope='function')
def random_workspace(tmpdir, random_workspace_name):
    os.environ[random_workspace_name] = tmpdir
    yield random_workspace_name
    del os.environ[random_workspace_name]
```
Then you can make use of this fixture everywhere without manually setup test workspace and delete environment variable manually. Kinda like setUp and tearDown in unittest.

Reply:
Good point!